pkg/agent/flowexporter/connections/conntrack_linux_test.go

```diff
@@ -151,11 +155,47 @@ func TestConnTackOvsAppCtl_DumpFlows(t *testing.T) {
 	}
 	mockOVSCtlClient.EXPECT().RunAppctlCmd("dpctl/dump-conntrack", false, "-m", "-s").Return(ovsctlCmdOutput, nil)
 
-	conns, err := connDumper.DumpFlows(uint16(openflow.CtZone))
+	conns, totalConns, err := connDumper.DumpFlows(uint16(openflow.CtZone))
 	if err != nil {
 		t.Errorf("conntrackNetdev.DumpConnections function returned error: %v", err)
 	}
 	assert.Equal(t, len(conns), 1)
 	assert.Equal(t, conns[0], expConn, "filtered connection and expected connection should be same")
+	assert.Equal(t, len(outputFlow), totalConns, "Number of connections in conntrack table should be equal to outputFlow")
+}
+
+func TestSetupConnTrackParameters(t *testing.T) {
+	err := setupConntrackParameters()
+	if err != nil {
+		t.Logf("Setup conntrack parameters function returned error: %v", err)
```

Comment:
how about using `t.Skipf("skipping test as trying to set up conntrack parameters returned an error: %v", err)` ?

Reply:
Sounds good, done.